users/jobs/daily/scrape.py

```diff
@@ -0,0 +1,132 @@
+from bs4 import BeautifulSoup
+import re
+import requests
+import psycopg2
+from django_extensions.management.jobs import DailyJob
+
+conn = psycopg2.connect(host="localhost", database="postgres",user="vikrant",password="password")
+c = conn.cursor()
+s = requests.Session()
+s.get("https://oa.cc.iitk.ac.in/Oa/Jsp/Main_Frameset.jsp")
+s.get("https://oa.cc.iitk.ac.in/Oa/Jsp/Main_Intro.jsp?frm='SRCH'")
+s.get("https://oa.cc.iitk.ac.in/Oa/Jsp/OAServices/IITK_Srch.jsp?typ=stud")
+
+headers = {
+    "Referer": "https://oa.cc.iitk.ac.in/Oa/Jsp/OAServices/IITK_Srch.jsp?typ=stud"
+}
+
+headers1 = {
+    "Referer": "https://oa.cc.iitk.ac.in/Oa/Jsp/OAServices/IITk_SrchStudRoll_new.jsp"
+}
+
+payload = {
+    'k4': 'oa',
+    'numtxt': '',
+    'recpos': 0,
+    'str': '',
+    'selstudrol': '',
+    'selstuddep': '',
+    'selstudnam': '',
+    'txrollno': '',
+    'Dept_Stud': '',
+    'selnam1': '',
+    'mail': ''
+}
+
+payload1 = {
+    'typ': ['stud'] * 12,
+    'numtxt': '',
+    'sbm': ['Y'] * 12
+}
+TOTAL = 8385
```

Comment:
I’m not sure if this is the total number of students now. It was the case a year ago. Have you checked whether this is the correct number?